sapi/phpdbg/tests/gh15210.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+GH-15210 use after free after continue
+--CREDITS--
+YuanchengJiang
```

Comment:
We don't really use credit sections any longer, as those were more to go around the limitations of SVN

Reply:
okidoki